src/css/toc.css

```
@@ -112,3 +112,19 @@
   line-height: 1.125;
   color: inherit;
 }
+
+.toc .toc-menu li.toc-H1 {
+  font-weight: bold;
+  font-size: 110%;
+}
+
+.toc .toc-menu li.toc-H3 {
+  list-style-type: circle;
```

Just a personal preference, but I don't really like the circle list style in the toc. Would you object to a hyphen?

```suggestion
  list-style-type: '-';
```

---

Also not sure why you've made heading 1 bold. There should only be one heading 1 in the page (the page title) and it shouldn't appear in the table of contents!

---

Yeah, that was just as the demo page on preview does have a h1, can remove.